tests/tamr_client/test_project.py

```diff
@@ -1,32 +1,24 @@
 import pytest
-import responses
 
 import tamr_client as tc
-from tests.tamr_client import utils
+from tests.tamr_client import fake, utils
 
 
-@responses.activate
+@fake.json
 def test_from_resource_id_mastering():
     s = utils.session()
     instance = utils.instance()
```

Comment:
Once we move the utilities into the `fake` module:

```suggestion
    s = fake.session()
    instance = fake.instance()
```